### PR TITLE
Lose the past

### PR DESCRIPTION
**Pull request by cclauss**
Let's do it the old fashioned way without the help of __six__ or __future__.  It is a bit more long winded but it works.

### CODE REVIEW

**Review by bmsdave — APPROVED** [review]

File: tools/testrunner/local/pool.py [hunk 1]
```diff
@@ -6,14 +6,17 @@
 # for py2/py3 compatibility
 from __future__ import print_function
 
-from Queue import Empty
 from contextlib import contextmanager
-from multiprocessing import Process, Queue
```
Comment (bmsdave) [inline tools/testrunner/local/pool.py:11]: As I understand it, we can use `multiprocessing` here, as it is compatible with both python 2 and python 3.

case with:
```
-from multiprocessing import Process, Queue
+try:
+  from queue import Empty  # Python 3
+except ImportError:
+  from Queue import Empty  # Python 2
```

gives error

```
ERROR: setUpClass (__main__.SystemTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "unittests/run_tests_test.py", line 155, in setUpClass
    pool.setup_testing()
  File "/opt/node/tmp/v8/tools/testrunner/local/pool.py", line 29, in setup_testing
    del Queue
NameError: global name 'Queue' is not defined
```

**Comment by cclauss** [comment]
[Read the comments](https://github.com/cclauss/v8-1/blob/master/tools/testrunner/local/pool.py#L37).  They are trying NOT to import multiprocessing but instead to _monkeypatch threading Queue to look like multiprocessing Queue_.  Does #3 fare better?

**Comment by bmsdave** [comment]
Ohh.. it helps. I create request review https://chromium-review.googlesource.com/c/v8/v8/+/1454477

**Comment by cclauss** [comment]
Awesome.  I changed the bug number to point to the right issue.

OPTIONAL: You might look at __flake8 . --select=E502__

**Pull request merged**